problems/removeEvenStrings.js

```diff
@@ -6,8 +6,38 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
-
+function removeEvenStrings(arr) {
+    console.log(arr)
+    let evenString = arr.filter((el) => {
+        console.log(el)
+        el % 2 === 0;
+        el++
```

Comment:
This is not helping us. filter will already iterate through all elements, we don't need to increment anything.